### PR TITLE
Fixed duplicate keys in accordion

### PR DESCRIPTION
**Pull request by gchi25**

### CODE REVIEW

File: test/pages/LifeEvent.spec.js [hunk 1]
```diff
@@ -271,16 +271,19 @@ describe("Life Event page", () => {
     })
     wrapper.vm.lifeEventBenefits = wrapper.vm.allLifeEventBenefits = [
       {
+        slug: "slugTwo",
```
Comment (danny-englander) [inline test/pages/LifeEvent.spec.js:274]: Is there any reason why `slugTwo` comes before `slugOne`?

File: test/pages/LifeEvent.spec.js [hunk 2]
```diff
@@ -271,16 +271,19 @@ describe("Life Event page", () => {
     })
```
Comment (danny-englander) [inline test/pages/LifeEvent.spec.js:271]: How does this file relate to `test/Components/Accordion.spec.js`? (If at all?)

**Review by danny-englander — COMMENTED** [review]
@gchi25 I just had a few questions here. Thank you.

File: test/pages/LifeEvent.spec.js (hunk 1 above)
Comment (gchi25) [reply test/pages/LifeEvent.spec.js:274]: No

File: test/pages/LifeEvent.spec.js (hunk 2 above)
Comment (gchi25) [reply test/pages/LifeEvent.spec.js:271]: Accordion is child component.
:key="`acc-key-${benefit.slug}`"
After adding slug, has key:
acc-key-slugTwo
acc-key-slugOne
acc-key-slugThree

**Pull request merged**